### ISSUE TITLE
Accessibility- Report Page- Activity: 1.3.4- Orientation- Tables and tab being cut off

### ISSUE THREAD

**Issue by aj-he** [archesproject/arches-her#1265]
When the report page is viewed in portrait mode, several tables are cut off as well as the JSON tab is unable to be selected

1.3.4- Orientation
Content is not restricted to one display orientation (Portrait or Landscape) and should be suited to the users' needs.
Many hand-held devices offer a mechanical switch or a system setting (or both) to allow the user to lock the device display to a specific orientation. Where a user decides to lock their entire device to an orientation, all applications are expected to pick up that setting and to display content accordingly.

![Image](https://github.com/user-attachments/assets/48df2b30-d007-4a97-9ea0-f487ae1331df)

![Image](https://github.com/user-attachments/assets/ff77832a-fa96-487f-9436-6afa6eb58892)

![Image](https://github.com/user-attachments/assets/c3be61af-c2c0-4f8d-95c4-2b7c77b28632)

**Comment by aj-he** [comment]
Already fixed so closing with agreement from tester
